code/parachain/runtime/picasso/src/governance.rs

```diff
@@ -78,6 +78,86 @@ parameter_types! {
 	// cspell:disable-next
 	pub const DemocracyId: LockIdentifier = *b"democrac";
 	pub RootOrigin: RuntimeOrigin = frame_system::RawOrigin::Root.into();
+	pub const AlarmInterval: BlockNumber = 1;
+}
+
+pallet_referenda::impl_tracksinfo_get!(TracksInfo, Balance, BlockNumber);
+impl pallet_referenda::Config for Runtime {
+	type RuntimeCall = RuntimeCall;
+
+	type RuntimeEvent = RuntimeEvent;
+
+	type WeightInfo = weights::referenda::WeightInfo<Self>;
+
+	type Scheduler = Scheduler;
+
+	type Currency = Balances;
+
+	type SubmitOrigin = frame_support::traits::EitherOf<
+		system::EnsureSignedBy<TechnicalCommitteeMembership, Self::AccountId>,
+		system::EnsureSignedBy<CouncilMembership, Self::AccountId>,
+	>;
+	#[cfg(not(feature = "fastnet"))]
+	type CancelOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "fastnet")]
+	type CancelOrigin = EnsureRootOrOneSixthNativeTechnical;
+
+	#[cfg(not(feature = "fastnet"))]
+	type KillOrigin = EnsureRootOrMoreThenHalfNativeCouncil;
+	#[cfg(feature = "fastnet")]
+	type KillOrigin = EnsureRootOrOneSixthNativeCouncil;
+
+	type Slash = ();
+
+	type Votes = pallet_conviction_voting::VotesOf<Runtime>;
+
+	type Tally = pallet_conviction_voting::TallyOf<Runtime>;
+
+	type SubmissionDeposit = ConstU128<1_000_000_000_000>;
```

Comment:
1 PICA? why not 100 PICA?

Reply:
previously we agreed to make this small to make easier for gov token holders